backend/corpora/api_server/app.py

```diff
@@ -174,7 +172,7 @@ def handle_corpora_error(exception):
     )
 
 
-@app.errorhandler(InternalServerError)
+@app.errorhandler(Exception)
```

Comment:
Isn't this also going to be triggered by exceptions that produce responses for the client, such as 400 responses, and which are not server errors? 🤔

Reply:
The 400 responses for invalid parameters are thrown by connexion which throws a ProblemException. We have a handler for ProblemExceptions above.

Reply:
We would be failing a bunch of unit tests if this wasn't working properly.

Reply:
Ok. As long as we're sure we aren't masking any non-500 errors with this.